envi/tests/test_arch_amd64.py

```diff
@@ -179,10 +179,10 @@ def test_envi_amd64_assorted_instrs(self):
     ('PSRLQ', '660FD3DC', 'psrlq xmm3,xmm4', 'psrlq xmm3,xmm4'),
     ('PSRLQ', '660F73d10f', 'psrlq xmm1,15', 'psrlq xmm1,15'),
     ('PSRLDQ (66)', '660f73d808', 'psrldq xmm0,8', 'psrldq xmm0,8'),
-    ('PSRLDQ (66)', '660f73b5aa4141', 'psllq xmm5,170', 'psllq xmm5,170'),
-    ('PSRLDQ (66)', '660f73f5aa4141', 'psllq xmm5,170', 'psllq xmm5,170'),
-    ('PSRLDQ (66)', '660f73b1aa4141', 'psllq xmm1,170', 'psllq xmm1,170'),
-    ('PSRLDQ (66)', '660f73b9aa4141', 'pslldq xmm1,170', 'pslldq xmm1,170'),
+    ('PSRLDQ (66)', '660f73b5aa', 'psllq xmm5,170', 'psllq xmm5,170'),
```

Comment:
ODAWEB suggests this is a bad encoding:
![image](https://user-images.githubusercontent.com/9836679/101282529-c63cc380-37a3-11eb-88ed-59cc05250806.png)

since it decodes to the same thing as the next test, i'm wondering myself.

Reply:
Oh yea. This was from my torturing of the disassemblers. It may be incorrect (I'm betting it probably is), but our disassembler allows it due to us not making an presumptions on the portions of the modrm byte that aren't used.

I can take it out since I forgot to earlier, but it is a good thing to note that we are slightly more permissive in certain cases.